subsys/bluetooth/host/l2cap.c

```diff
@@ -921,6 +926,115 @@ __weak void bt_test_l2cap_data_pull_spy(struct bt_conn *conn,
 {
 }
 
+__maybe_unused static bool user_data_not_empty(const struct net_buf *buf)
+{
+	size_t ud_len = sizeof(struct closure);
```

Comment:
I know this is only used for `struct closure` currently, but since the function name is more generic, why not use `buf->user_data_size`?